### PR TITLE
mockingbird v0.0.1 release

### LINKED ISSUE #8: release mockingbird v0.0.1
- [x] *npm compile fix*: webpack bundle is being generated, yet, there are some minor issues while importing the code on another shadow-cljs+helix project
- [x] *target.css*: need to check if any difference has occurred in the now used file and the one from source at compile time, since for now, the css code generated from mockingbird is only copied to user project once (or when this file deleted)

### PR DESCRIPTION
**Pull request by Felipe-gsilva**
fixes #8 

## Release Notes

- **New Features**
  - Introduced a new `LICENSE` file specifying the MIT License terms.
  - Added documentation for using the `mockingbird-cljs` NPM package, including installation and usage examples.
  - New `nav-link` function created for modular navigation links in the navbar.
  - Enhanced `message` component to display the current date.
  - Introduced a centralized module for layout components in `mockingbird.layout.core`.

- **Improvements**
  - Updated `README.md` for clarity on npm usage and prerequisites.
  - Restructured `shadow-cljs.edn` to include a new build configuration.
  - Improved error handling in various components and functions.
  - Updated `footer` component to accept dynamic content.

- **Bug Fixes**
  - Adjusted installation instructions in `README.md` to include missing dependencies.

- **Chores**
  - Cleaned up `package.json` by removing duplicate dependencies and updating scripts.
  - Removed outdated `webpack.config.js` and `docs/Roadmap.md` files.

### CODE REVIEW

**Comment by J0sueTM** [comment]
@coderabbitai configuration

File: package.json [hunk 1]
```diff
@@ -1,18 +1,22 @@
 {
   "name": "mockingbird-lib",
   "version": "0.0.1",
-  "main": "dist/mockingbird-lib.js",
-  "files": [  
+  "main": "dist/mockingbird.components.core.js",
```
[the remark this answers is not in the document]
Comment (Felipe-gsilva) [reply package.json:4]: the file is not found because it is ignored by .gitignore

**Comment by avelino** [comment]
@coderabbitai review

**Comment by Felipe-gsilva** [comment]
@coderabbitai show config

**Comment by avelino** [comment]
🎉

**Pull request merged**